core-api/src/main/java/com/sequenceiq/cloudbreak/api/model/v2/AmbariV2Request.java

```diff
@@ -75,14 +75,18 @@
     private ConfigStrategy configStrategy = ConfigStrategy.ALWAYS_APPLY_DONT_OVERRIDE_CUSTOM_VALUES;
 
     @ApiModelProperty(ModelDescriptions.ClusterModelDescription.BLUEPRINT_INPUTS)
-    private Set<BlueprintInputJson> blueprintInputs = new HashSet<>();
+    private Set<BlueprintInputJson> blueprintInputs;
 
     @ApiModelProperty(ModelDescriptions.ClusterModelDescription.BLUEPRINT_CUSTOM_PROPERTIES)
     private String blueprintCustomProperties;
 
     @ApiModelProperty(ModelDescriptions.ClusterModelDescription.CONNECTED_CLUSTER)
     private ConnectedClusterRequest connectedCluster;
 
+    public AmbariV2Request() {
+        blueprintInputs = new HashSet<>();
```

Comment:
is there any reason why we initializing here the hashset? I think we should follow the same way as we do it in other domain objects or using this way in any other domain